package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "snek-dancer",
+  "version": "1.0.0",
+  "description": "### `Are we snek? or are we dancer?` -- A python based incrementer game for meme and learning purposes",
```

Comment:
What does the ### do here?

Reply:
Good question. I just 'yes'd' the default params. It should be safe to remove though, right?